include/mbedtls/build_info.h

```diff
@@ -177,6 +177,35 @@
 #define MBEDTLS_PK_PARSE_C
 #endif
 
+/* Internal helper to define which fields in the pk_context structure should
```

Comment:
Ah, I'd rather keep that comment (hence the associated `#define`) in `pk.h` where it's more discoverable - it's a bit hidden here. It's OK for simpler things to be a bit less discoverable (in order to avoid duplication by putting them in a more central place), but this one's really non-trivial and quite important for the implementation of PK.

Could we change the definition of `MBEDTLS_PK_HAVE_ECC_KEYS` so that it doesn't depend on this one? I think `ECP_C || (USE_PSA && WANT_KEY_TYPE_ECC)` would work. Wdyt?